test/system/users_test.rb

```diff
@@ -578,4 +578,22 @@ class UsersTest < ApplicationSystemTestCase
     user = users(:hajime)
     assert_match(/#{user.id}\.png$/, img.native['src'])
   end
+
+  test 'mentor can see retired and hibernated tabs' do
+    visit_with_auth '/users', 'mentormentaro'
+    assert_link '休会', href: '/users?target=hibernated'
+    assert_link '退会', href: '/users?target=retired'
+  end
+
+  test 'admin can see retired and hibernated tabs' do
+    visit_with_auth '/users', 'komagata'
+    assert_link '休会', href: '/users?target=hibernated'
+    assert_link '退会', href: '/users?target=retired'
+  end
+
+  test 'if the user is not admin or mentor, the user cannot see retired and hibernated tabs' do
```

Comment:
```suggestion
  test 'user can not see retired and hibernated tabs if the user is not admin or mentor' do
```

こちらの方が英語的にも、他と統一する意味でもわかりやすいかな〜と思いました。

Reply:
修正しました！